numpy/core/src/common/simd/vsx/arithmetic.h

```diff
@@ -118,6 +118,25 @@
 
 // Horizontal add: Calculates the sum of all vector elements.
 
+NPY_FINLINE npy_uint32 npyv_sum_u8(npyv_u8 a)
+{
+    const npyv_u32 zero4 = npyv_zero_u32();
+    npyv_u32 sum4 = vec_sum4s(a, zero4);
+    return (npy_uint32)vec_extract(vec_sums(sum4, zero4), 3);
```

Comment:
The [first CI error](https://travis-ci.com/github/numpy/numpy/jobs/473632623#L4110) suggest something is off with `vec_sums`. That is [documented](http://cdn.openpowerfoundation.org/wp-content/uploads/resources/leabi/content/dbdoclet.50655245_56497.html) as `vector signed int vec_sums (vector signed int, vector signed int)` but here they are `npy_uint32` so is this a signed/unsigned problem?
```
numpy/core/src/common/simd/vsx/arithmetic.h: In function ‘npyv_sum_u8’:

  numpy/core/src/common/simd/vsx/arithmetic.h:125:5: error: invalid parameter combination for AltiVec intrinsic __builtin_vec_sums

       return (npy_uint32)vec_extract(vec_sums(sum4, zero4), 3);

       ^~~~~~

  numpy/core/src/common/simd/vsx/arithmetic.h:125:5: error: invalid parameter combination for AltiVec intrinsic 
```

Reply:
yes indeed, casting to `npyv_s32` will fix it.